### PR TITLE
chore: Update helm chart name

### PR DESCRIPTION
**Pull request by felixwang9817**
**What this PR does / why we need it**: Note that I did not remove the existing `feast-python-server` helm chart. Instead, I simply made a new `feast-feature-server` helm chart that copies `feast-python-server`. 

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: infra/charts/feast-feature-server/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+```
+helm install feast-python-server . --set image.repository=REPO --set image.tag=TAG
+```
+
+Deployment assumes that `feature_store.yaml` exists on docker image. Example docker image:
+```
+FROM python:3.8
+
+RUN apt update && \
+    apt install -y jq
```
Comment (achals) [inline infra/charts/feast-feature-server/README.md:52]: Need to install arrow here?

File: infra/charts/feast-feature-server/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "feast-python-server.name" . }}
+  labels:
+    {{- include "feast-python-server.labels" . | nindent 4 }}
```
Comment (achals) [inline infra/charts/feast-feature-server/templates/service.yaml]: Do these names need to change?

**Review by achals — APPROVED** [review]

File: infra/charts/feast-feature-server/templates/service.yaml (hunk 2 above)
Comment (felixwang9817) [reply infra/charts/feast-feature-server/templates/service.yaml]: yup fixed

File: infra/charts/feast-feature-server/README.md (hunk 1 above)
Comment (felixwang9817) [reply infra/charts/feast-feature-server/README.md:52]: yes good call

Comment (felixwang9817) [reply infra/charts/feast-feature-server/README.md:52]: added

**Review by achals — APPROVED** [review]
/lgtm

**Pull request merged**